### PR TITLE
Add r-mratios.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

File: recipes/r-mratios/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  summary: With this package, it is possible to perform (simultaneous) inferences for ratios
+    of linear combinations of coefficients in the general linear model. In particular,
+    tests and confidence interval estimations for ratios of treatment means in the normal
+    one-way layout and confidence interval estimations like in (multiple) slope ratio
+    and parallel line assays can be carried out. Moreover, it is possible to calculate
+    the sample sizes required in comparisons with a control based on relative margins.
+    For the simple two-sample problem, functions for a t-test for ratio-formatted hypotheses
+    and the corresponding Fieller-type confidence interval are provided assuming homogeneous
+    or heterogeneous group variances.
+  license_family: GPL
```
Comment (jdblischak) [inline recipes/r-mratios/meta.yaml]: Please add the GPL-3 license.

Comment (cbrueffer) [reply recipes/r-mratios/meta.yaml]: The problem is that the package specifies the license as GPL, not more explicitly GPL2 or GPL3; it package also doesn't include a license file. We've had this problem with other packages in the past, e.g. https://github.com/conda-forge/r-heatmap.plus-feedstock

Comment (jdblischak) [reply recipes/r-mratios/meta.yaml]: When a package only specifies "GPL", CRAN expands this to "GPL2 | GPL3".

https://cran.r-project.org/package=mratios

Thus I think we should add either the GPL2 or GPL3 license file.

Also, @ocefpaf [recommends](https://github.com/conda-forge/staged-recipes/pull/3984#issuecomment-330966741) writing `license: GPL-2.0 and GPL-3.0` in this situation, but I don't feel strongly about the `license` field.

Comment (cbrueffer) [reply recipes/r-mratios/meta.yaml]: OK, I've gone the "GPL2 and GPL3" route.

**Pull request merged**